code/modules/mob/living/carbon/xenomorph/castes/behemoth/abilities_behemoth.dm

```diff
@@ -1443,3 +1370,77 @@
 /datum/ammo/xeno/earth_pillar/landslide/on_hit_obj(obj/hit_object, obj/projectile/proj)
 	. = ..()
 	return on_hit_anything(get_turf(hit_object), proj)
+
+
+// ***************************************
+// *********** Global Procs
+// ***************************************
+#define AREA_ATTACK_DAMAGE_MECHA_MODIFIER 10
+
+/**
+ * Checks for any atoms caught in the attack's range, and applies several effects based on the atom's type.
+ * * turfs_to_attack: The turfs affected by this proc.
+ * * enhanced: Whether this is enhanced or not.
+ * * instant: Whether this is done instantly or not.
+*/
+/proc/behemoth_area_attack(owner, list/turf/affected_turfs, enhanced, damage_multiplier = 1.5, paralyze_duration = 0.8 SECONDS)
```

Comment:
why does this need to be a global proc?

Reply:
Because this is used across various abilities. I decided to make it a global instead since it was easier to adjust for each case and saved me a lot of ability references.